### PR TITLE
Try to productise docker-compose setup

### PR DESCRIPTION
**Pull request by pudo**
Would this be OK for you @alexmojaki ?

### CODE REVIEW

**Comment by alexmojaki** [comment]
Doesn't seem to work:

```
➜  opensanctions git:(pudo/fixup) make shell
docker-compose up -d postgres redis
Starting opensanctions_postgres_1 ... done
Starting opensanctions_redis_1    ... done
sleep 3
docker-compose run worker sh
Starting opensanctions_postgres_1 ... done
Starting opensanctions_redis_1    ... done
/memorious # memorious run un_sc_sanctions
Traceback (most recent call last):
  File "/usr/bin/memorious", line 11, in <module>
    load_entry_point('memorious', 'console_scripts', 'memorious')()
  File "/memorious/memorious/cli.py", line 110, in main
    cli(obj={})
  File "/usr/lib/python3.7/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python3.7/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python3.7/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib/python3.7/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python3.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/memorious/memorious/cli.py", line 48, in run
    TaskRunner.run_sync()
  File "/memorious/memorious/task_runner.py", line 58, in run_sync
    for task in Queue.tasks(stop_on_timeout=True):
  File "/memorious/memorious/model/queue.py", line 27, in tasks
    yield (job_stage.operation, state, load_json(data))
AttributeError: 'JobStage' object has no attribute 'operation'
/memorious # memorious run us_ofac
Traceback (most recent call last):
  File "/usr/bin/memorious", line 11, in <module>
    load_entry_point('memorious', 'console_scripts', 'memorious')()
  File "/memorious/memorious/cli.py", line 110, in main
    cli(obj={})
  File "/usr/lib/python3.7/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python3.7/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python3.7/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib/python3.7/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python3.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/memorious/memorious/cli.py", line 48, in run
    TaskRunner.run_sync()
  File "/memorious/memorious/task_runner.py", line 58, in run_sync
    for task in Queue.tasks(stop_on_timeout=True):
  File "/memorious/memorious/model/queue.py", line 27, in tasks
    yield (job_stage.operation, state, load_json(data))
AttributeError: 'JobStage' object has no attribute 'operation'
```

File: docker-compose.yml [hunk 1]
```diff
@@ -11,35 +8,39 @@ services:
...
   postgres:
     image: postgres:11.4
     volumes:
-      - local_postgres_data:/var/lib/postgresql/data
-    env_file:
-      - envfile.env
+      - "./data/postgres-data:/var/lib/postgresql/data"
+    environment:
+      POSTGRES_USER: opensanctions
+      POSTGRES_PASSWORD: opensanctions
```
Comment (alexmojaki) [inline docker-compose.yml]: Need to add `POSTGRES_DB: opensanctions`

**Comment by alexmojaki** [comment]
Otherwise this seems to work now, thank you! If we could merge it that would be great.

File: docker-compose.yml (hunk 1 above)
Comment (pudo) [reply docker-compose.yml]: It just defaults to the username if no DB name is given, which seems OK here.

**Pull request merged**